components/developer/maven/manifests/sample-manifest.p5m

```diff
@@ -27,40 +27,44 @@ file path=usr/share/maven/NOTICE
 file path=usr/share/maven/README.txt
 file path=usr/share/maven/bin/m2.conf
 file path=usr/share/maven/bin/mvn
-file path=usr/share/maven/bin/mvn.bat
+file path=usr/share/maven/bin/mvn.cmd
 file path=usr/share/maven/bin/mvnDebug
-file path=usr/share/maven/bin/mvnDebug.bat
+file path=usr/share/maven/bin/mvnDebug.cmd
 file path=usr/share/maven/bin/mvnyjp
-file path=usr/share/maven/boot/plexus-classworlds-2.5.1.jar
+file path=usr/share/maven/boot/plexus-classworlds-2.5.2.jar
 file path=usr/share/maven/conf/logging/simplelogger.properties
 file path=usr/share/maven/conf/settings.xml
-file path=usr/share/maven/lib/aether-api-0.9.0.M2.jar
-file path=usr/share/maven/lib/aether-api.license
-file path=usr/share/maven/lib/aether-connector-wagon-0.9.0.M2.jar
-file path=usr/share/maven/lib/aether-connector-wagon.license
-file path=usr/share/maven/lib/aether-impl-0.9.0.M2.jar
-file path=usr/share/maven/lib/aether-impl.license
-file path=usr/share/maven/lib/aether-spi-0.9.0.M2.jar
-file path=usr/share/maven/lib/aether-spi.license
-file path=usr/share/maven/lib/aether-util-0.9.0.M2.jar
-file path=usr/share/maven/lib/aether-util.license
+file path=usr/share/maven/conf/toolchains.xml
 file path=usr/share/maven/lib/aopalliance-1.0.jar
 file path=usr/share/maven/lib/cdi-api-1.0.jar
 file path=usr/share/maven/lib/cdi-api.license
-file path=usr/share/maven/lib/commons-cli-1.2.jar
-file path=usr/share/maven/lib/commons-io-2.2.jar
-file path=usr/share/maven/lib/commons-lang-2.6.jar
+file path=usr/share/maven/lib/commons-cli-1.4.jar
+file path=usr/share/maven/lib/commons-cli.license
+file path=usr/share/maven/lib/commons-io-2.5.jar
+file path=usr/share/maven/lib/commons-io.license
+file path=usr/share/maven/lib/commons-lang3-3.5.jar
+file path=usr/share/maven/lib/commons-lang3.license
 file path=usr/share/maven/lib/ext/README.txt
-file path=usr/share/maven/lib/guava-14.0.1.jar
+file path=usr/share/maven/lib/guava-20.0.jar
+file path=usr/share/maven/lib/guice-4.0-no_aop.jar
+file path=usr/share/maven/lib/jansi-1.17.jar
+file path=usr/share/maven/lib/jansi-native/README.txt
+file path=usr/share/maven/lib/jansi-native/freebsd32/libjansi.so
+file path=usr/share/maven/lib/jansi-native/freebsd64/libjansi.so
+file path=usr/share/maven/lib/jansi-native/linux32/libjansi.so
+file path=usr/share/maven/lib/jansi-native/linux64/libjansi.so
+file path=usr/share/maven/lib/jansi-native/osx/libjansi.jnilib
+file path=usr/share/maven/lib/jansi-native/windows32/jansi.dll
+file path=usr/share/maven/lib/jansi-native/windows64/jansi.dll
```

Comment:
All the jansi-native files must be removed; maybe we need to provide our own for oi?